src/validationErrorFormatter.js

```diff
@@ -1,15 +1,83 @@
 var RDF = require('../includes/Erics_RDF.js');
 
 function formatError(fail) {
-    var rule;
-    //if(fail.rule.valueClass._ === "ValueSet")
-    //    rule = RDF.Triple(fail.rule.label, fail.rule.nameClass.term, fail.rule.valueClass.values[0].term);
-    //else
-    //    rule = RDF.Triple(fail.rule.label, fail.rule.nameClass.term, fail.rule.valueClass.type);
+    
     return {
         name: fail._,
         triple : fail,
-        req_lev: fail.rule.req_lev
+        req_lev: fail.rule.req_lev,
+        toString: function() {
```

Comment:
would probably opt for this function being called and the result being passed ie.
`description: errorToString(fail)`